### PR TITLE
Add correct method for hardcoded `/f/{fileid}`

### PR DESCRIPTION
**Pull request by nickvergessen**
Regression from nextcloud/server#14626
* Try to load any `/f/{fileid}` link

### CODE REVIEW

**Comment by nickvergessen** [comment]
Same wrong hack (route name method not matching the invoked method) was used in talk:
https://github.com/nextcloud/spreed/pull/1612

**Review by rullzer — APPROVED** [review]
:see_no_evil:

File: apps/files/lib/Controller/ViewController.php [hunk 1]
```diff
@@ -128,6 +128,18 @@ protected function getStorageInfo() {
...
 	}
 
+	/**
+	 * @NoCSRFRequired
+	 * @NoAdminRequired
+	 *
+	 * @param string $fileid
+	 * @return TemplateResponse|RedirectResponse
+	 */
+	public function showFile(string $fileid = null): Response {
```
Comment (MorrisJobke) [inline apps/files/lib/Controller/ViewController.php]: Bildschirmfoto 2019-03-14 um 16 27 07


🙈  Let me fix that.

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Review by ChristophWurst — APPROVED** [review]
:elephant:

**Comment by MorrisJobke** [comment]
> * tests/acceptance/features/apps.feature:83

"fixed" in #14681

**Pull request merged**